types/node-telegram-bot-api/index.d.ts

```diff
@@ -490,6 +490,7 @@ declare namespace TelegramBot {
         last_name?: string | undefined;
         username?: string | undefined;
         language_code?: string | undefined;
+        is_premium?: boolean | undefined;
```

Comment:
Should the type be `true | undefined`? The documentation you linked to (https://core.telegram.org/bots/api#user) says the type of this property is `True` as opposed to `Boolean`.